file.go

```diff
@@ -334,9 +276,6 @@ func (f *File) InsertAt(p0 int, s []rune) {
 	if len(s) != 0 {
 		f.Modded()
 	}
```

Comment:
You have to call `oeb.inserted()` here right?

Reply:
I was calling it within oeb, but I'll move it here